lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -69,6 +75,13 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
 
             line = "* #{message} via #{name} (@#{username})"
             changelog_sections[section].push(line)
+          elsif items.size == 0
+            UI.important("Cannot find pull request associated to #{sha}. Using commit information and adding it to the Other section")
+            message = commit["commit"]["message"]
+            name = commit["commit"]["author"]["name"]
+            username = commit["author"]["login"]
+            line = "* #{message} via #{name} (@#{username})"
+            changelog_sections[:other].push(line)
```

Comment:
let's add a slack warning so we _know_ when this is happening, since we're assuming that this is a patch change but we don't have a way of asserting it

Reply:
Do you mind if I add this in another PR? I think we have RN will also get affected by this bug (I pushed to main by error the other day https://github.com/RevenueCat/react-native-purchases/commit/32309f49903848b432abdfab1bfaf425f42cb76a). We don't have any slack notification from fastlane yet and I worry it will take me time to setup and test